apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/nfts/components/nft-property.tsx

```diff
@@ -8,22 +7,19 @@ interface NftPropertyProps {
 
 export const NftProperty: React.FC<NftPropertyProps> = ({ property }) => {
   return (
-    <Card as={Flex} flexDir="column" gap={2}>
-      {property?.trait_type && (
-        <Text
-          size="label.sm"
-          color="primary.500"
-          textAlign="center"
-          lineHeight={1.2}
-        >
-          {property?.trait_type}
-        </Text>
-      )}
-      <Text size="label.md" textAlign="center">
-        {typeof property?.value === "object"
-          ? JSON.stringify(property?.value || {})
-          : property?.value}
-      </Text>
+    <Card className="flex p-3">
+      <div className="m-auto flex flex-col items-center">
+        {property?.trait_type && (
+          <p className="text-center text-primary text-sm leading-[1.2]">
+            {property?.trait_type}
```

Comment:
`text-primary` is not a valid foreground color - please check the output and verify

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/VtULqGKSp33n0VHOzNJK/f405abc8-bd7a-4b81-8f6e-482821a30fec.png)

Somehow text-primary gives me the blue-ish color that I need. But I have updated to `text-foreground-link` as you said, since it looks better

Reply:
The `--primary` is for background only.  Refer to shadcn themeing guide : https://ui.shadcn.com/docs/theming